### ISSUE TITLE
SCSS compile error with scssphp 1.5.0

### ISSUE THREAD

**Issue by malberts** [ProfessionalWiki/Bootstrap#49]
A new version of scssphp was released today: https://github.com/scssphp/scssphp/releases/tag/v1.5.0
With the previous version this issue does not happen: https://github.com/scssphp/scssphp/releases/tag/v1.4.1

When using scssphp v1.5.0 with:
* MediaWiki: 1.35.2
* SCSS library: 2.0.5
* Bootstrap extension: 4.5.0 (= Bootstrap library 4.6.0)
* Chameleon skin: master

This might be a [SCSS library](https://github.com/ProfessionalWiki/SCSS) issue instead, but I'm logging it here since the error mentions a problem with Bootstrap.

When loading any page the CSS is not loaded:
![image](https://user-images.githubusercontent.com/1428594/118332722-d1420f80-b50a-11eb-9a73-ddb95066d814.png)

When loading the page with `?debug=true` an error can be seen in the Bootstrap/Chameleon CSS:
![image](https://user-images.githubusercontent.com/1428594/118332803-f9317300-b50a-11eb-963e-bae29fc425b1.png)
Response:
```
/* SCSS compile error: $n: List index may not be 0.: extensions/Bootstrap/resources/bootstrap/scss/mixins/_box-shadow.scss on line 13, at column 9
Call Stack:
#0 mixin box-shadow extensions/Bootstrap/resources/bootstrap/scss/_custom-forms.scss on line 36
#1 import extensions/Bootstrap/resources/bootstrap/scss/_custom-forms.scss (unknown file) on line 1*/
```

**Comment by JeroenDeDauw** [comment]
.... again?! This is the 4th or 5th time that a new release of scssphp breaks things... sigh. Maybe we should just pin it to a specific version.

This issue is a bit misplaced, since scssphp is included in https://github.com/ProfessionalWiki/SCSS. Then again, I'd like to get rid of the SCSS extension. In version 4.3.x of the Bootsrtap extension I tried bundling it. Came pretty close to getting it to work but forgot about something and ended up revering. Perhaps something you could have a look at.

**Comment by s7eph4n** [comment]
There is also a php module available for SCSS that should be less problematic. As far as I understand it uses (or is compiled from?) the original SCSS lib, i.e. not a re-implementation. It's not a standard PHP extension, obviously, so should not be the default, but it might be an idea to include it as an alternative for people who have control over their server and/or are concerned for performance.

**Comment by malberts** [comment]
Closing, because the original issue has been resolved in the latest scssphp 1.5.2.
